lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelBatchHydrator.kt

```diff
@@ -131,11 +132,15 @@ internal class NadelBatchHydrator(
                 .map { actorQuery ->
                     async { // This async executes the batches in parallel i.e. executes hydration as Deferred/Future
                         val hydrationSourceService = executionBlueprint.getServiceOwning(instruction.location)!!
+                        val hydrationActorField =
+                            FieldCoordinates.coordinates(instruction.actorFieldContainer, instruction.actorFieldDef)
+
                         val serviceHydrationDetails = ServiceExecutionHydrationDetails(
-                            instruction.timeout,
-                            instruction.batchSize,
-                            hydrationSourceService,
-                            instruction.location
+                            timeout = instruction.timeout,
+                            batchSize = instruction.batchSize,
+                            hydrationSourceService = hydrationSourceService,
+                            hydrationSourceField = instruction.location,
+                            hydrationActorField = hydrationActorField
```

Comment:
again expanded it to have named args

Reply:
The ServiceCallback  now will have the extra field in the `ServiceExecutionHydrationDetails` object